atlas/AtlasMl/atlasml/routers/competency.py

```
@@ -89,25 +94,50 @@ async def get_competency(id: str):
+    return Response(
+        status_code=status.HTTP_200_OK,
+        content=b"[]",
+        media_type="application/json",
+    )
+
+
+@router.post(
+    "/save", response_model=list[Competency], dependencies=[Depends(TokenValidator())]
+)
+async def save_competencies(request: SaveCompetencyRequest):
+    # TODO: @ArdaKaraman call required pipeline with the input you do not need to return anything
```

Ruff (0.11.9)

Line too long (97 > 88)

(E501)